src/pages/Bridge/IssueForm/index.tsx

```diff
@@ -110,8 +110,7 @@ const IssueForm = (): JSX.Element | null => {
   const { bridgeLoaded, bitcoinHeight, btcRelayHeight, parachainStatus } = useSelector(
     (state: StoreType) => state.general
   );
-
-  const { governanceTokenBalanceLoading, governanceTokenBalance } = useGovernanceTokenBalance();
+  const { isLoading: isBalancesLoading, data: balances } = useGetBalances();
```

Comment:
I noticed you did `data: balances` in multiple places, how about returning the data directly in `balances` property?

Reply:
That makes sense, so let's follow this pattern from now on :+1: